src/popup/components/TransactionListItem.vue

```diff
@@ -108,10 +109,13 @@ export default defineComponent({
     let timerInterval: NodeJS.Timer;
     const transactionDate = ref();
 
-    const currentTransaction = computed(
-      () => (props.multisigTransaction || props.transaction),
+    const currentTransaction = computed<ITransaction>(
+      () => (props.multisigTransaction as ITransaction || props.transaction),
     );
 
+    // temp if protocol undefined assume it is aeternity
+    const transactionProtocol = computed(() => props.transaction?.protocol ?? PROTOCOLS.aeternity);
```

Comment:
Maybe instead of this just hide the elements if the `props.transaction` is falsy with the `v-if`?

Reply:
This is supposed to only check for `props.trsansaction.protocol` and uses aeternity if it is missing.
All transactions have a protocol assigned to them, but sometimes aeternity transactions do not have a protocol value